app/code/Magento/Sales/Controller/Adminhtml/Order/CreditmemoLoader.php

```diff
@@ -181,7 +181,12 @@ public function load()
         $creditmemoId = $this->getCreditmemoId();
         $orderId = $this->getOrderId();
         if ($creditmemoId) {
-            $creditmemo = $this->creditmemoRepository->get($creditmemoId);
+            try{
+                $creditmemo = $this->creditmemoRepository->get($creditmemoId);
+            }catch(\Exception $e){
+                $this->messageManager->addErrorMessage(__('This creditmemo no longer exists.'));
+                return false;
+            }
```

Comment:
You don't need this modification.
If there's `NoSuchEntityException` thrown by `->get()`, then you should let it pass to the method that needs it: ( app/code/Magento/Sales/Controller/Adminhtml/Order/Creditmemo/View.php ) and handle it there.

Reply:
In this case, we get fail testPhpStan Magento\Framework\Exception\LocalizedException